docs/ADD_NEW_DATA.md

```diff
@@ -0,0 +1,12 @@
+# How to add and use new test dataset
```

Comment:
Do we want to have this here? I guess this is fine to be in `master` (to inform everyone how to add new data) but not really necessary in your branch?

Reply:
ok, I've seen it other branches also, that's why I've added it. I'll remove the additional files then.